scripts/generate_pstokes_run.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+"""
+Pipeline script for generating pstokes visibility.
+"""
+
+from hera_pspec import pstokes
+from pyuvdata import UVData
+import pyuvdata
+
+ap = pstokes.generate_pstokes_argparser()
+args = ap.parse_args()
+uvd = UVData()
+uvd.read(args.inputdata)
+if args.outputdata is None:
+    args.outputdata = args.inputdata
+if args.inplace:
+    # if inplace, append new pstokes onto existing file.
+    uvd_output = copy.deepcopy(uvd)
+else:
+    # otherwise, output uvd does not contain original polarizations.
+    uvd_output = pstokes.construct_pstokes(uvd, uvd, args.pstokes[0])
+for p in args.pstokes:
+    if pyuvdata.utils.polstr2num(p) not in uvd_output.polarization_array:
+        uvd_output += pstokes.construct_pstokes(uvd, uvd, pstokes=p)
+
+# overwrite file.
```

Comment:
the read and write have different arguments, so perhaps this comment should just be: write the file.

also I don't think inplace is really the correct kwarg here, b/c in the normal use-case, the outputdata is not the inputdata, so you are writing a new file, not overwriting the original file (the latter of which we never do in our pipeline anyways by policy, I think).

b/c it has a specific use-case of either keeping the vispols or not, perhaps its better to use something like `keep_vispols`?

Reply:
Good point. I will change `inplace` to `keep_vispols`.